components/button/stories/button.stories.js

```diff
@@ -286,3 +295,18 @@ WithWrapping.args = {
 WithWrapping.parameters = {
 	chromatic: { disableSnapshot: true },
 };
+
+/**
+ * The normal behavior for lengthy text in the given horizontal space available is that it will wrap to form another line. By using the `noWrap` option, the lengthy button text will horizontally expand the entirety of the container. Be sure to define the width of the button container using this option. Please note: this can cause undesired overflow experiences and to help prevent this, the overflowing text will attempt to hide by showing an ellipsis (...). 
+ * */
+
+export const DisableWrapping = TextWrapTemplate.bind({});
```

Comment:
What do you think about showing a third option which demonstrates that this can escape its container? I experimented in the inspector a little bit:
![Screenshot 2024-09-19 at 3 10 34 PM](https://github.com/user-attachments/assets/db7cc670-5162-40fc-97ec-149f79149806)
Here I'm setting the width on the inner container (adjustable through `wrapperStyles`), and the buttons with ellipsis are `max-inline-size: 100%`.

This makes me wonder if we need a new mod for `max-inline-size`, if we're expecting users in SWC to be able to want to change that property too (@pfulton)?